src/app.py

```
@@ -23,38 +26,128 @@
         return jsonify({"error": "حدث خطأ أثناء تحميل المكتبة"})
     # ---
-    json_result = event(titles, tst_prnt_all=False) or {}
+    start_time = time.time()
+    # ---
+    json_result, no_labs = event(titles, return_no_labs=True, tst_prnt_all=False) or {}
+    # ---
+    len_result = len(json_result)
     # ---
-    return jsonify(json_result)
+    for x in no_labs:
+        if x not in json_result.keys():
```

Ruff (0.8.2)

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)